centreon/src/Core/Media/Domain/Model/Media.php

```diff
@@ -30,21 +30,21 @@ class Media
 {
     private string $className;
 
-    private ?string $comment = null;
-
     /**
      * @param int $id
      * @param string $filename
      * @param string $directory
+     * @param string|null $comment
      * @param string|null $data
      *
-     * @throws \Assert\AssertionFailedException
+     * @throws AssertionFailedException
      */
     public function __construct(
         readonly private int $id,
         private string $filename,
         private string $directory,
-        private ?string $data
+        private ?string $comment,
+        readonly private ?string $data
     ) {
         $this->className = (new \ReflectionClass($this))->getShortName();
```

Comment:
Why is `$this->className` a property, because it has no use outside th constructor ?